commit 4b58b392ca971d7307575eb09648adb552a053e8
Author: Kieran Bingham <kieran.bingham@ideasonboard.com>
Date:   Mon Mar 18 17:16:41 2019 +0000

    vcsm: Remove set but unused variable
    
    The 'success' variable is set by the call to vchi_service_close() but never checked.
    Remove it, keeping the call in place.
    
    Signed-off-by: Kieran Bingham <kieran.bingham@ideasonboard.com>

diff --git a/drivers/char/broadcom/vc_sm/vc_vchi_sm.c b/drivers/char/broadcom/vc_sm/vc_vchi_sm.c
index f8b909a09adb..ca4ec40bb5fb 100644
--- a/drivers/char/broadcom/vc_sm/vc_vchi_sm.c
+++ b/drivers/char/broadcom/vc_sm/vc_vchi_sm.c
@@ -361,11 +361,9 @@ int vc_vchi_sm_stop(struct sm_instance **handle)
 
 	/* Close all VCHI service connections */
 	for (i = 0; i < instance->num_connections; i++) {
-		int32_t success;
-
 		vchi_service_use(instance->vchi_handle[i]);
 
-		success = vchi_service_close(instance->vchi_handle[i]);
+		vchi_service_close(instance->vchi_handle[i]);
 	}
 
 	kfree(instance);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1524_4b58b392ca971d7307575eb09648adb552a053e8.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1524_4b58b392ca971d7307575eb09648adb552a053e8.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
